src/client/app/+home/home.component.ts

```diff
@@ -1,12 +1,12 @@
 import { FORM_DIRECTIVES } from '@angular/common';
 import { Component } from '@angular/core';
 
-import { NameListService } from '../../shared/index';
+import { NameListService } from '../shared/index';
 
 @Component({
   selector: 'sd-home',
-  templateUrl: 'app/+home/components/home.component.html',
-  styleUrls: ['app/+home/components/home.component.css'],
+  templateUrl: 'app/+home/home.component.html',
```

Comment:
Yes, our configuration is commonjs specific so we can. In case developers switch to any other module format they have to manually handle the migration (such as __moduleName for system).